### DISCUSSION TITLE
Does this plugin accept Swift toolchains for Linux?

### DISCUSSION THREAD

**Discussion by alicerunsonfedora** [noctule-ide/noctule-swift-plugin#33] [Q&A]
I'm using a Linux environment via WSL, but I notice that the plugin doesn't accept my Swift toolchain path. Is this plugin known to work with Linux/WSL this way, or does it only accept macOS Swift toolchains?

![Screenshot of the toolchains section in the Swift plugin settings for IntelliJ](https://github.com/jansorg/swift-plugin/assets/13445064/a9578c35-9165-45bc-96a2-15f3caa286c3)

**Comment by jansorg** [comment]
I had tested this on macOS, Linux and Windows, but not yet on WSL. I'll take s look at this setup...
